tests/protractor/page-objects/forms/delivery-report.po.js

```diff
@@ -0,0 +1,1238 @@
+        </item>
+        <item>
+          <label ref="jr:itext('/delivery/group_note/default_chw_sms/miscarriage:label')"/>
+          <value>miscarriage</value>
+        </item>
+        <item>
+          <label ref="jr:itext('/delivery/group_note/default_chw_sms/unknown:label')"/>
+          <value>unknown</value>
+        </item>
+        <item>
+          <label ref="jr:itext('/delivery/group_note/default_chw_sms/anc_only_facility_birth:label')"/>
+          <value>anc_only_facility_birth</value>
+        </item>
+        <item>
+          <label ref="jr:itext('/delivery/group_note/default_chw_sms/anc_only_home_sba_birth:label')"/>
+          <value>anc_only_home_sba_birth</value>
+        </item>
+        <item>
+          <label ref="jr:itext('/delivery/group_note/default_chw_sms/anc_only_home_no_sba_birth:label')"/>
+          <value>anc_only_home_no_sba_birth</value>
+        </item>
+        <item>
+          <label ref="jr:itext('/delivery/group_note/default_chw_sms/anc_only_facility_stillbirth:label')"/>
+          <value>anc_only_facility_stillbirth</value>
+        </item>
+        <item>
+          <label ref="jr:itext('/delivery/group_note/default_chw_sms/anc_only_home_sba_stillbirth:label')"/>
+          <value>anc_only_home_sba_stillbirth</value>
+        </item>
+        <item>
+          <label ref="jr:itext('/delivery/group_note/default_chw_sms/anc_only_home_no_sba_stillbirth:label')"/>
+          <value>anc_only_home_no_sba_stillbirth</value>
+        </item>
+        <item>
+          <label ref="jr:itext('/delivery/group_note/default_chw_sms/anc_only_miscarriage:label')"/>
+          <value>anc_only_miscarriage</value>
+        </item>
+      </select1>
+      <select1 ref="/delivery/group_note/is_sms_edited">
+        <label ref="jr:itext('/delivery/group_note/is_sms_edited:label')"/>
+        <item>
+          <label ref="jr:itext('/delivery/group_note/is_sms_edited/yes:label')"/>
+          <value>yes</value>
+        </item>
+        <item>
+          <label ref="jr:itext('/delivery/group_note/is_sms_edited/no:label')"/>
+          <value>no</value>
+        </item>
+      </select1>
+      <input appearance="multiline" ref="/delivery/group_note/g_chw_sms">
+        <label ref="jr:itext('/delivery/group_note/g_chw_sms:label')"/>
+        <hint ref="jr:itext('/delivery/group_note/g_chw_sms:hint')"/>
+      </input>
+    </group>
+    <group appearance="field-list summary" ref="/delivery/group_summary">
+      <input ref="/delivery/group_summary/submit">
+        <label ref="jr:itext('/delivery/group_summary/submit:label')"/>
+      </input>
+      <input appearance="h1 yellow" ref="/delivery/group_summary/r_summary">
+        <label ref="jr:itext('/delivery/group_summary/r_summary:label')"/>
+      </input>
+      <input appearance="h4 center" ref="/delivery/group_summary/r_patient_info">
+        <label ref="jr:itext('/delivery/group_summary/r_patient_info:label')"/>
+      </input>
+     
```

Comment:
I would move the hospital and person documents below back into the test. The only thing this PO cares about is the form.